assertk-js/src/test/kotlin/test/assertk/assertions/ComparableJsTest.kt

```diff
@@ -0,0 +1,26 @@
+package test.assertk.assertions
+
+import assertk.assert
+import assertk.assertions.isCloseTo
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFails
+
+class ComparableJvmTest {
+    @Test
+    fun isCloseToFloat_with_delta_out_of_range_fails() {
+        val error = assertFails {
+            assert(10.1f).isCloseTo(15.0f, 3f)
+        }
+        assertEquals("expected <10.1> to be close to <15> with delta of <3>, but was not", error.message)
```

Comment:
If you want to put this in the common module you could do
```kotlin
assertEquals("expected ${show(10.1f)} to be close to ${show(15.0f)} with delta of ${show(3f)}, but was not", error.message)
```

Reply:
We cannot move this into common. As the error messages are platform specific.
For example
```kotlin
assert(10.1f).isCloseTo(15.0f, 3.5f)
```
jvm will show error message as (with `f` for float)
```kotlin
expected <10.1f> to be close to <15.0f> with delta of <3.5f>, but was not
```
js will show error message as (without `f` for float and `15.0` will be displayed as `15`)
```kotlin
expected <10.1> to be close to <15> with delta of <3.5>, but was not
```
So I put the failure tests in jvm and js.

Reply:
by putting `show()` in the expected message, it should convert it to the platform-specific display.

Reply:
Yeah, I agree. I feel it is better to use `show` while showing an error message.
As we are using `show`, we have to write test specific to platform.

Am i right? Am i missing something?

Reply:
Sorry, think we are talking past each other. What I'm trying to say is you can put the test in the common module by using show. Ex:
```kotlin
@Test
fun isCloseToFloat_with_delta_out_of_range_fails() {
    val error = assertFails {
        assert(10.1f).isCloseTo(15.0f, 3f)
    }
    assertEquals("expected ${show(10.1f)} to be close to ${show(15.0f)} with delta of ${show(3f)}, but was not", error.message)
}
```

Reply:
Sorry. got it now. Moved tests into common module. Thanks